templates/AureliaSpa/ClientApp/app/components/fetchdata/fetchdata.ts

```diff
@@ -0,0 +1,27 @@
+/// <reference path="../../../../node_modules/aurelia-fetch-client/doc/whatwg-fetch.d.ts" />
+/// <reference path="../../../../node_modules/aurelia-fetch-client/doc/url.d.ts" />
```

Comment:
As far as we know, there's nothing incorrect in our dts. It's just that we use the Fetch spec which relies on the there being dts for fetch and url. Those should be provided by TS because they are standards, but they aren't, so you have to include your own. If there's something we need to fix on our side, we're happy to do it though.